src/classes/User.as

```diff
@@ -478,7 +558,8 @@ package classes
                             "miss": scoreResults[3],
                             "boo": scoreResults[4],
                             "maxcombo": scoreResults[5],
-                            "rawscore": ((scoreResults[0] * 50) + (scoreResults[1] * 25) + (scoreResults[2] * 5) - (scoreResults[3] * 10) - (scoreResults[4] * 5))};
+                            "rawscore": ((scoreResults[0] * 50) + (scoreResults[1] * 25) + (scoreResults[2] * 5) - (scoreResults[3] * 10) - (scoreResults[4] * 5)),
+                            "equiv": Number(0.00)};
```

Comment:
`"equiv": Number(0.00)` isn't really needed, just `"equiv": 0` is enough.

Reply:
So that's what I had originally but it was doing a string comparison during sorting, for example:
50.34
49.46
4.45
34.36
34.00
26.00
2.56
15.03

rather than
50.34
49.46
34.36
34.00
26.00
15.03
4.45
2.56

if you want it to be handled/cast during the comparison instead I could maybe do that, but seemed easier just to cast it there (admittedly I guess I didn't need to do 0.00 and probably could do 0)?

Reply:
Must have been the sort that was the problem, this is fixed to just 0 now.